sr_robot_launch/scripts/teach_mode_node

```diff
@@ -50,33 +58,51 @@ class TeachModeManager:
         self.modes = ["TRAJECTORY_MODE", "TEACH_MODE", "POSITION_MODE", "DIRECT_PWM_MODE"]
 
     @staticmethod
-    def _create_hand_joints():
+    def _create_hand_joints(waiting_timeout: int = 60) -> Dict[str, list]:
```

Comment:
```suggestion
    def _create_hand_joints(waiting_timeout: Union[int, float] = 60) -> Dict[str, list]:
```
I guess that makes more sense since timeout can be a float